tools/run_tests/performance/bq_upload_result.py

```diff
@@ -157,6 +173,42 @@ def _populate_metadata_inplace(scenario_result):
     scenario_result['metadata'] = metadata
 
 
+def _populate_metadata_from_file(scenario_result, test_metadata_file):
+    with open(test_metadata_file, 'r') as f:
```

Comment:
There is an assumption here that the file always exists. Should there be tests, for instance, only try to read the file if test_metadata_file is not empty? I suppose we can require that the file is readable if the variable is set.

Reply:
Yes, I will update this part, I was assuming the check was done in run.sh, as we decide to upload the data with or without metadata I will add a check here.